fsharp-backend/src/BackendOnlyStdLib/HttpClient.fs

```diff
@@ -131,20 +138,22 @@ let makeHttpCall
         req.Version <- System.Net.HttpVersion.Version20
 
         // username and password - note that an actual auth header will overwrite this
+        // WHATISTHIS
```

Comment:
I believe I'm simply unfamiliar with uri.UserInfo. I'll research here and follow up.